### PR TITLE
Get rid of a lot of Returns

### PR DESCRIPTION
**Pull request by kshyatt**
this is what you get for getting me started

### CODE REVIEW

File: base/linalg/svd.jl [hunk 1]
```diff
@@ -170,16 +170,40 @@ end
 """
     svdvals!(A)
 
-Returns the singular values of `A`, saving space by overwriting the input.
-See also [`svdvals`](@ref).
+Return the singular values of `A`, saving space by overwriting the input.
```
Comment (fredrikekre) [inline base/linalg/svd.jl]: From #24719?

**Review by fredrikekre — APPROVED** [review]
:smile:

**Comment by andyferris** [comment]
I'm curious - what's the linguistic reasoning behind this?

(There's also Construct vs Constructs and other such examples in the first sentence of docstrings)

**Comment by fredrikekre** [comment]
From the style guide (point 2) https://docs.julialang.org/en/latest/manual/documentation/#Documentation-1

> The one-line sentence should use the imperative form ("Do this", "Return that") instead of the third person (do not write "Returns the length...") when documenting functions.

**Pull request merged**